### PR TITLE
Updating documentation and updating to new Unity version

### PR DESCRIPTION
**Pull request by JeffSkyrunner**

### CODE REVIEW

File: ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.15f1
-m_EditorVersionWithRevision: 2019.4.15f1 (fbf367ac14e9)
+m_EditorVersion: 2019.4.17f1
+m_EditorVersionWithRevision: 2019.4.17f1 (667c8606c536)
```
Comment (dagguh) [inline ProjectSettings/ProjectVersion.txt:2]: Nice!
Could you also update the link in the docs? b093f572fc6fb3fa08f69e1f8c28d49c4aacd26f

Comment (JeffSkyrunner) [reply ProjectSettings/ProjectVersion.txt:2]: Done... I hope :)

Comment (dagguh) [reply ProjectSettings/ProjectVersion.txt:2]: Seems you opened a new PR for that #2026. I can stitch them together if you want 🙂

Comment (JeffSkyrunner) [reply ProjectSettings/ProjectVersion.txt:2]: Yes please, I missclicked :)

**Comment by dagguh** [comment]
Closed in favor of https://github.com/dagguh/data-hunt/pull/2026

**Pull request closed without merging**